src/IK/include/BipedalLocomotion/IK/AngularMomentumTask.h

```diff
@@ -0,0 +1,129 @@
+/**
+ * @file AngularMomentumTask.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_IK_ANGULAR_MOMENTUM_TASK_H
+#define BIPEDAL_LOCOMOTION_IK_ANGULAR_MOMENTUM_TASK_H
+
+#include <memory>
+
+#include <BipedalLocomotion/IK/IKLinearTask.h>
+
+#include <iDynTree/KinDynComputations.h>
+
+namespace BipedalLocomotion
+{
+namespace IK
+{
+
+/**
+ * AngularMomentumTask is a concrete implementation of the IK::Task. Please use this element if you
+ * want to control the position and the orientation of a given frame rigidly attached to the robot.
+ * The task assumes perfect control of the robot velocity \f$\dot{\nu}\f$ that contains the base
+ * linear and angular velocity expressed in mixed representation and the joint velocities.
+ * The task represents the following equation
+ * \f[
+ * J_\omega \nu = \omega ^ *
+ * \f]
+ * where \f$J_\omega\f$ is the angular part of the robot Jacobian and \f$\omega ^ *\f$ is the
+ * desired velocity. The desired velocity is chosen such that the orientation of the frame will
+ * asymptotically converge to the desired trajectory. \f$\omega ^ *\f$ is computed with a
+ * Proportional controller in \f$SO(3)\f$.
+ * @note Please refer to https://github.com/dic-iit/lie-group-controllers if you are interested in
```

Comment:
```suggestion
 * @note Please refer to https://github.com/ami-iit/lie-group-controllers if you are interested in
```

Reply:
Also this one is a copy-paste. I fixed the documentation